tuning/trainercontroller/callback.py

```diff
@@ -425,6 +432,20 @@ def on_step_end(
         kwargs["control"] = control
         self._actions_on_event(event_name="on_step_end", **kwargs)
 
+    def on_save(
+        self,
+        args: TrainingArguments,
+        state: TrainerState,
+        control: TrainerControl,
+        **kwargs,
+    ):
+        # Training arguments, state and controls are folded into kwargs to be passed off to
+        # handlers
+        kwargs["args"] = args
+        kwargs["state"] = state
+        kwargs["control"] = control
```

Comment:
Do we need to construct a new kwargs for this? Isn't just passing them in _actions_on_event just enough?